src/lib/updates/avatar.js

```diff
@@ -35,5 +36,10 @@ const uploadAvatars = async (lastUpdate, prevVersion, log) => {
       await userStorage.removeAvatar(true)
     }
   }
+
+  // cleanup feed to re-upload other user avatars during feed pages loading
+  // reuploading all the avatars cached in the feed here could take a lot of time
+  // so let users would do it by themselves time by time while scrolling
+  await AsyncStorage.removeItem('GD_feed')
```

Comment:
this wont help
1. avatar is fetched when reading receipt from blockchain
2. if you cache avatar while displaying it would happen anyways